doc/nrf/gs_ins_windows.rst

```diff
@@ -340,26 +340,59 @@ Complete the following steps to set up the |SES| environment:
 
        Open nRF Connect SDK Project menu
 
-#. Set the Zephyr Base directory to the full path to ``ncs\zephyr``.
-   The GNU ARM Embedded Toolchain directory is the directory where you installed the toolchain (for example, ``c:\gnuarmemb``).
+#. Set the Zephyr Base directory to the full path to ``ncs/zephyr``.
+   The GNU ARM Embedded Toolchain directory is the directory where you installed the toolchain (for example, ``c:/gnuarmemb``).
 
     .. figure:: images/ses_notset.png
        :alt: Zephyr Base Not Set prompt
 
        Zephyr Base Not Set prompt
 
+.. buildenv_end
+
+.. buildenv_path_start
+
+4. Make sure the locations of tools are added to the PATH variable.
+   On Windows and Linux, SES uses the PATH variable to find executables.
+
+.. buildenv_path_end
+
+.. _build_environment_settings_changes_win:
+
+.. buildenv_settings_changes_intro_start
+
+Changing the SES environment settings
+-------------------------------------
+
+If you want to change the SES environment settings later, click :guilabel:`Tools` -> :guilabel:`Options` and select the :guilabel:`nRF Connect` tab, as shown on the following screenshot from the Windows installation.
 
-If you want to change these settings later, click :guilabel:`Tools` -> :guilabel:`Options` and select the :guilabel:`nRF Connect` tab (see :ref:`ses_options_figure`).
+.. buildenv_settings_changes_intro_end
 
+.. _ses_options_figure:
+
+.. figure:: images/ses_options.png
+     :alt: nRF Connect SDK options in SES on Windows
+
+     nRF Connect SDK options in SES (Windows)
+
+.. buildenv_settings_changes_ctd_start
+
+Changing the SES environment settings might require modifying the PATH variable.
```

Comment:
No, these are independent. I *think* (not sure - @tejlmand do you know?) that the nRF Connect SDK options in SES have priority over the PATH. So if you configure a tool in SES, that is the one that is used, no matter what is in the PATH. But you cannot configure all tools in there, that's why we also need the PATH ...
But if you change any configuration in SES, you don't need to change anything in the PATH.

Reply:
> So if you configure a tool in SES, that is the one that is used, no matter what is in the PATH. 

Correct.

> But you cannot configure all tools in there, that's why we also need the PATH ...

Also correct. Ideally it should be possible to specify any tool in the SES, but as we cannot control if CMake or python suddenly call out for yet another tool, it is only possible to specify those that are common to give issues if not in correct version.
Secondly, being able to specify all tools, could give a too long list.